### ISSUE TITLE
Parser error?

### ISSUE THREAD

**Issue by athos** [clojureverse/clojurians-log-app#48]
Hi, there 😃 

I ran into an error when trying to access the following page:

- https://clojurians-log.clojureverse.org/clojurescript/2018-11-13

The error looks as follows:

<img width="500" alt="2018-11-15 14 09 03" src="https://user-images.githubusercontent.com/27441/48534203-ac916080-e8ea-11e8-9ea7-88f49f5916b1.png">

I'm suspecting this post on that day is causing it, which includes some strikethrough in a quote.

<img width="500" alt="2018-11-15 15 27 12" src="https://user-images.githubusercontent.com/27441/48534408-612b8200-e8eb-11e8-8fca-326ab79b9050.png">

**Comment by plexus** [comment]
This page is still erroring out.

**Comment by plexus** [comment]
@dawran6 how would you feel about trying your hand at this one. At the point where each message gets parsed and converted from markdown to hiccup we should wrap it in a try/catch, and in the catch just return the message unparsed, plus put a `println`, "Failed to parse message ..."

**Comment by dawran6** [comment]
Absolutely, I'm working on this. Is `println` the current preferred way of logging?

**Comment by plexus** [comment]
Yeah println is fine, we log the stdout of the process. I don't think there's any more proper logging in place yet

**Comment by plexus** [comment]
Pages that errored before according to google search console:

https://gist.github.com/90d310374bcdf234bfe4ea5cfb84da1d

Now that we no longer fail completely on parser errors we can look at those pages, find the messages that don't render, and add test cases for them (and then fix the parser :) )
